test/katex-spec.js

```diff
@@ -2386,6 +2386,23 @@ describe("A smash builder", function() {
     });
 });
 
+describe("A document fragment", function() {
+    it("should have paddings applied inside an extensible arrow", function() {
+        const markup = katex.renderToString("\\tiny\\xrightarrow\\textcolor{red}{x}");
+        expect(markup).toContain("x-arrow-pad");
```

Comment:
These should maybe be snapshot tests so it's easier to see the structure of the markup.

Reply:
@kevinbarabash But the markup result is too complicated, with a lot of unnecessary parts.

Reply:
Makes sense, thanks.